File: src/kyu_8/name_on_billboard/csharp/name_on_billboard_test.cs
namespace Solution {
  using NUnit.Framework;
  using System;
  [TestFixture]
  public class SolutionTest
  {
    [Test]
    public void SampleTests()
    {
      Assertion(600, "Jeong-Ho Aristotelis");
      Assertion(40,"CODEWARS", 5);
    }
    
    private static void Assertion(double expected, string name, double price = -1) =>
      Assert.AreEqual(
          expected,
          price == -1 ? Kata.Billboard(name) : Kata.Billboard(name, price),
      
          0.000001,
      
          $"\n  Name: \"{name}\"\n" +
            $"  Price: {(price == -1 ? 30 : price)}\n\n"
      
      );
  }
}

File: src/kyu_8/name_on_billboard/python/name_on_billboard_test.py
import codewars_test as test
from solution import billboard

sample_test_cases = [
    #  name                price  expected
    (('Jeong-Ho Aristotelis',  ),   600),
    (('Abishai Charalampos',   ),   570),
    (('Idwal Augustin',        ),   420),
    (('Hadufuns John',       20),   260),
    (('Zoroaster Donnchadh',   ),   570),
    (('Claude Miljenko',       ),   450),
    (('Werner Vigi',         15),   165),
    (('Anani Fridumar',        ),   420),
    (('Paolo Oli',             ),   270),
    (('Hjalmar Liupold',     40),   600),
    (('Simon Eadwulf',         ),   390),
]

@test.describe('Sample tests')
def sample_tests():
    for args, expected in sample_test_cases:
        @test.it(f'billboard({", ".join(map(repr, args))})')
        def _():
            test.assert_equals(billboard(*args), expected)